card/mod/machines/lib/javascript/decko_upload.js.coffee

```diff
@@ -5,7 +5,7 @@ $.extend decko,
 # only the file, type_id and attachment_card_name are needed
 # attachment_card_name is the original card name,
 # ex: card[subcards][+logo][image], card[file]
-    $(fileupload).bind 'fileuploadsubmit', (e,data) ->
+    $(fileupload).on 'fileuploadsubmit', (e,data) ->
```

Comment:
wow, was that hard to find?

Reply:
not really, jquery has a changelog for that